contrib/local.mk.example

```diff
@@ -2,7 +2,7 @@
 # Individual entries must be uncommented to take effect.
 
 # By default, the installation prefix is '/usr/local'.
-# CMAKE_EXTRA_FLAGS += -DCMAKE_INSTALL_PREFIX=/usr/local/nvim-latest
```

Comment:
Well, it's no strict requirement, but according to [this](https://cmake.org/cmake/help/v3.0/prop_cache/TYPE.html), it helps cmake GUIs provide valid entries (a file chooser in this case) to the user.

Thought it wouldn't hurt. :-) Especially now where the user doesn't have to use it directly anymore.